src/thunderbots/software/util/canvas_messenger/canvas_messenger.h

```diff
@@ -0,0 +1,84 @@
+#pragma once
```

Comment:
Could we add a quick start-of-file comment here?

Reply:
Done